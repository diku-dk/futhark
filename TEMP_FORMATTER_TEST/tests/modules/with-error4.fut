-- Cannot refine a type twice.
-- ==
-- error: not an abstract type

module type mt = {type t}

module type mt' = mt with t = i64

module type mt'' = mt' with t = i64
