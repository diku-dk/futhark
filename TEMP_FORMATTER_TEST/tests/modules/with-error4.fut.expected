1d077fa428daaa8efae8947f9e9146a3
